lib/index.js

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as bedrock from '@bedrock/core';
+import {agent} from '@bedrock/https-agent';
+import assert from 'assert-plus';
+import {httpClient} from '@digitalbazaar/http-client';
+import './config.js';
+
+const {config, util: {BedrockError}} = bedrock;
+
+export async function verify({token, remoteIp, client = 'default'}) {
```

Comment:
Improves error readability when people forget params:

```suggestion
export async function verify({token, remoteIp, client = 'default'} = {}) {
```